elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -28,61 +28,38 @@ ul.pl-order-blocks-connected-sortable li:hover{
 }
 
 li.ui-state-highlight {
-  /*This styling applies to the placeholder answer tile that appears when the user is dragging */
+  /* This styling applies to the placeholder answer tile created by jquery-ui when the user is dragging */
   border: none !important;
   list-style-type: none;
   background-color: #90a4ae !important;
   min-width: 3em !important;
 }
 
-ul.header {
-  color: black;
-  list-style-type: none;
-  margin-bottom: 0px;
-}
-
-ul.inline li {
-  display: inline-block;
-  margin-right: 5px;
-}
```

Comment:
Its time to get rid of this 'inline' option altogether. I'm almost positive that no one is using it because (1) its not documented anywhere, and (2) its been halfway broken for as long as I can remember and no one has complained. 

And actually, removing it won't completely break any questions that were using it, they will just be formatted slightly differently.